Cargo.toml

```diff
@@ -6,35 +6,25 @@ edition = "2021"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-esp-backtrace = { version = "0.14.1", features = [
+esp-backtrace = { version = "0.15.0", features = [
     "esp32c3",
     "exception-handler",
     "panic-handler",
     "println",
 ]}
 
-esp-hal = { version = "0.22.0", features = ["esp32c3"] }
-esp-println = { version = "0.12.0", features = ["esp32c3", "log"] }
+esp-hal = { version = "0.23.1", features = ["esp32c3"] }
+esp-println = { version = "0.13.0", features = ["esp32c3", "log"] }
 log = { version = "0.4.21" }
-esp-alloc = { version = "0.5.0" }
+esp-alloc = { version = "0.6.0" }
 embedded-io = "0.6.1"
 shtcx = "1.0.0"
 
 embedded-io-async = "0.6.1"
-embassy-net = { version = "0.5.0", features = [ "tcp", "udp", "dhcpv4", "medium-ethernet"] }
+embassy-net = { version = "0.6.0", features = [ "tcp", "udp", "dhcpv4", "medium-ethernet"] }
 
-esp-wifi = { version = "0.11.0", default-features = false, features = ["esp32c3", "utils", "wifi", "esp-alloc", "log"] }
+esp-wifi = { version = "0.12.0", default-features = false, features = ["esp32c3", "utils", "wifi", "esp-alloc", "log"] }
 heapless = { version = "0.8.0", default-features = false }
-smoltcp_011 = { version = "0.11.0", default-features = false, features = [
-    "medium-ethernet",
-    "proto-dhcpv4",
-    "proto-ipv4",
-    "socket-dhcpv4",
-    "socket-icmp",
-    "socket-raw",
-    "socket-tcp",
-    "socket-udp",
-], package = "smoltcp" }
 smoltcp_012 = { version = "0.12.0", default-features = false, features = [
```

Comment:
I guess we could also consider calling this one just `smolltcp` now, right?

Reply:
Sure